.github/workflows/build-docs.yml

```diff
@@ -0,0 +1,27 @@
+name: Build Docs
+on:
+  push:
```

Comment:
Run only when [specific files are changed](https://github.com/Sage-Bionetworks/sage-monorepo/blob/333301ed6c3db53574a02989ce393b7fd258ce1c/.github/workflows/build-devcontainer-image.yml#L6-L9) to save compute resources?

Reply:
Good idea

Reply:
I specified that the CI should only trigger if there are changes to the mkdocs config file, any of the markdown files in `docs/` or any file in `docs/images/`